cpp/csp/python/Conversions.h

```diff
@@ -666,6 +666,30 @@ inline Dictionary fromPython( PyObject * o )
     return out;
 }
 
+template<>
+inline std::vector<Dictionary> fromPython(PyObject* o)
```

Comment:
Should this be its own PR incase we need to revert something from the Dynamic WS Connections?

Reply:
It's actually not needed anymore, I can just remove (I was using it for parsing raw python objects in the cpp side inside an adapter but changed that)